README.md

```diff
@@ -72,59 +72,106 @@ ws.onmessage = (message) => {
+  It can be an absolute block number, or a negative value, meaning how many blocks from the current head block on the chain.
+  Ex: -2500 means 2500 blocks in the past, relative to the head block.
+- `fetch` **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)** Whether to fetch an initial snapshot of the requested entity.
+- `with_progress` **[number](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Number)** Frequency of the progress of blocks processing (within the scope of a req_id).
+  You will, at a maximum, receive one notification each 250 milliseconds (when processing large amounts of blocks),
+  and when blockNum % frequency == 0. When you receive a progress notification associated with a stream (again, identified by its req_id),
+  you are guaranteed to have seen all messages produced by that stream, between the previous progress notification and the one received (inclusively).
 
 ### get_actions
 
 Get Actions
 
 #### Parameters
 
-- `account` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Account
-- `action_name` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Action Name
-- `receiver` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)?** Receiver
-- `options` (optional, default `{}`)
+- `account` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** Contract account targeted by the action.
+- `data` **[object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** Data Options (optional, default `{}`)
+  - `data.receiver` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)?** Specify the receiving account executing its smart contract.
+    If left blank, defaults to the same value as `account`.
+  - `data.action_name` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)?** Name of the action called within the account contract.
+  - `data.with_ramops` **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)?** Stream RAM billing changes and reasons for costs of storage produced by each action.
+  - `data.with_inline_traces` **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)?** Stream the inline actions produced by each action.
+  - `data.with_deferred` **[boolean](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Boolean)?** Stream the modifications to deferred transactions produced by each action.
+- `options` **[OptionalParams](#optionalparams)** Optional Parameters (optional, default `{}`)
+
+#### Examples
+
+```javascript
+ws.send(get_actions("eosio.token", { action_name: "transfer" }))
```

Comment:
```suggestion
ws.send(get_actions("eosio.token", "transfer"))
```

Reply:
I've updated the examples and changed the params slightly.

Only two params now (`data parameters` & `optional parameters`) both as defined as Objects

## Examples

**get_table_rows**

```ts
ws.send(get_table_rows({code: "eosio", scope: "eosio", table_name: "global"}));
```

**get_actions**

```ts
ws.send(get_actions({account: "eosio.token", action_name: "transfer"}));
```